common/isolationgroup/default-isolation-group-state.go

```diff
@@ -52,22 +68,61 @@ type defaultIsolationGroupStateHandler struct {
 	subscriptions map[string]map[string]chan<- ChangeEvent
 }
 
+const (
+	DefaultIsolationGroupConfigStoreManagerGlobalMapping dynamicconfig.MapKey = iota
```

Comment:
The key should not be exported. And why do we use MapKey?

Reply:
Regarding the export, good call, let me fix, 

regarding the mapkey, my possibly flawed memory was that it is because the value encoded actually is a map (`echo eyJOYW1lIjoiem9uZS0xIiwiU3RhdGUiOjJ9 | base64 -D
{"Name":"zone-1","State":2}`) but let me verify